README.md

```diff
@@ -144,12 +144,9 @@ The user must specify the input directory containing the files to run the simula
 
 ## Running multiple simulations in parallel
 
-Up until now, all simulations were run synchronously which gives feedback while the simulation is running until it finishes. However, this is not always the best option, for example, when launching two simulations it implies that they run one after the other. This
-is not optimal for when users need to launch hundreds of simulations.
+**Inductiva API** allows you to launch multiple simulation in parallel. After a simulation is submitted, the client unblocks - simulations run asynchronously - and can run other tasks. Users are not limited to wait for the simulation to finish.
```

Comment:
Users don't know what the client is, so other option: "After submitting a simulation, users are free to perform other tasks since simulations run independently in the background."?

Reply:
Wonderfull!